lib/active_model/serializer.rb

```diff
@@ -105,7 +105,7 @@ def self.root_name
       name.demodulize.underscore.sub(/_serializer$/, '') if name
     end
 
-    attr_accessor :object, :root, :meta, :meta_key, :scope
+    attr_accessor :object, :root, :meta, :meta_key, :links, :scope
```

Comment:
Hi @leandrocp,  from my point of view the keys `first`, `prev`, `next` and `last`should be resposability of pagination and no one should override it. Thus, the others keys won't be replaced because I am doing [update](https://github.com/rails-api/active_model_serializers/pull/1041/files#diff-ae1e38e96a8bf28ae37918a65c448c73R171). So, besides the keys above every key pass through controllers will continue.